packages/tangle/timemanager.go

```diff
@@ -32,7 +33,15 @@ type TimeManager struct {
 
 	lastConfirmedMutex   sync.RWMutex
 	lastConfirmedMessage LastConfirmedMessage
+	lastSyncedMutex      sync.RWMutex
 	lastSynced           bool
+
+	mu             sync.Mutex
+	ticker         *time.Ticker
+	started        typeutils.AtomicBool
+	stopped        typeutils.AtomicBool
```

Comment:
Are `started` and `stopped` actually used anywhere?

Reply:
I've removed them.